velox/exec/HashBuild.cpp

```diff
@@ -1052,6 +1052,13 @@ void HashBuild::reclaim(
   TestValue::adjust("facebook::velox::exec::HashBuild::reclaim", this);
 
   if (exceededMaxSpillLevelLimit_) {
+    const auto* config = spillConfig();
```

Comment:
Shall we check this in canReclaim()? We can check if exceededMaxSpillLevelLimit_ or not by checking the operator's runtime stats